apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -347,6 +367,8 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
     */
     function canClose(uint256 _voteId) external view returns (bool) {
         Vote storage vote_ = _getVote(_voteId);
+        // Note: do we check for isExecuted() because the original call to close may fail (but not revert) for some reason?
+        // Note: wondering if we should expose a permissionless method to close any cancelled vote (and that way we can also set their status to be cancelled)?
```

Comment:
These are minor, but curious what you think :).

Reply:
I see the benefit ofc, but I wouldn't add it to be honest. Users can go through the agreement app instead. I wouldn't add unnecessary methods to the interface simply because we are already close to the limit.